pkgs/by-name/je/jetuml/package.nix

```
@@ -0,0 +1,67 @@
+{
+  stdenvNoCC,
+  lib,
+  makeDesktopItem,
+  fetchurl,
+  jdk,
+  makeWrapper,
+}:
+
+let
+  jdkWithFX = jdk.override { enableJavaFX = true; };
+in
+stdenvNoCC.mkDerivation rec {
+
+  pname = "jetuml";
+  version = "3.9";
+
+  src = fetchurl {
+    url = "https://github.com/prmr/JetUML/releases/download/v${version}/JetUML-${version}.jar";
+    hash = "sha256-wACGbHeRQ5rXcuI1J3eTfQraWp8eWtkIAPo7BNGcFUU=";
+  };
+
+  dontUnpack = true;
+
+  buildInputs = [
+    jdkWithFX
+    makeWrapper
+  ];
+
+  desktopItem = makeDesktopItem {
+    name = "jetuml";
+    desktopName = "JetUML";
+    genericName = "UML Tool";
+    categories = [
+      "Application"
+      "Development"
+      "ProjectManagement"
+    ];
+    icon = "jet";
+    exec = "jetuml";
+    comment = meta.description;
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/java $out/share/icons
+
+    cp -r ${desktopItem}/share/applications $out/share/
+
+    jar xf $src jet.png
+    cp jet.png $out/share/icons/jet.png
+
+    ln -s $src $out/share/java/JetUML-${version}.jar
```

Shouldn't make a symlink to the source cause that will just add an unnecessary dependency. 
```suggestion
    install -Dm444 $src $out/share/java/JetUML-${version}.jar
```